### DISCUSSION TITLE
Create index of names in markdown?

### DISCUSSION THREAD

**Discussion by abunavas** [community/community#24006] [New to GitHub]
<p>Hi,<br>
Any way to create index of names in markdown (and export to pdf with pandoc)<br>
I don’t know if this is markdown or pandoc question, sorry…</p>

**Comment by ernest-phillips** [comment]
<p>Hi <a class="mention" href="/abunavas">@abunavas</a> :wave:</p>
<p>Could you explain in a bit more detail what is meant by “index of names” ?<br>
What does this look like and what are you trying to accomplish with it?</p>
<p>Depending on your code editor, you may be able to install an extension to convert your markdown files to a PDF.<br>
Here is one such example from the VIsual Studio Code marketplace:<br>
<a href="https://marketplace.visualstudio.com/items?itemName=yzane.markdown-pdf" class="onebox" target="_blank" rel="noopener">https://marketplace.visualstudio.com/items?itemName=yzane.markdown-pdf</a></p>

**Comment by abunavas** [comment]
<p>Hi, thank you for answer. By index of names, I mean list of personal names mentioned in text, with numbers of pages they appear. I am using Sublime Text, and can covert .md to PDF with pandoc, but I need to have index at the end.</p>
<p>Like in this picture</p>
<p><img src="https://user-images.githubusercontent.com/11981078/181080604-63c96927-f6a3-48f8-8ec6-6d790996a4a1.jpeg" alt="" data-base62-sha1="hVCcXvXS7cQhVFG6pnPNMbIWwrr" width="500" height="410"></p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Thank you for that additional context.</p>
<p>There is no concept of columns in a markdown file. The closest workaround I can think of is creating a table. It’s not very fun to implement, but these are the limitations of the Markdown language.</p>
<p>eg:</p>
<pre><code class="lang-auto">| A                     | &lt;span style="font-weight:normal"&gt;Eyre, Jane, 88&lt;/span&gt; |
| :-------------------- | :---------------  |
| Airey, Dawn, XIII-XIV p.248 &lt;br&gt; Angelou, Maya, 71 &lt;br&gt; Armstrong, Neil, 71|**F**   |
|                               |  Fiennes, Ralph, 72 &lt;br&gt; Fitzgerald, F. Scott, 120-121, 197            |
</code></pre>
<p>That would yield something like this:<br>
<img src="https://user-images.githubusercontent.com/5134583/181080605-03a00acd-0328-447a-bf7d-86185d922fdc.png" alt="Screen Shot 2021-03-05 at 5.42.16 PM" data-base62-sha1="v3WZF5fsWlBcsEULrPxgJnvwsDJ" width="450" height="184"></p>

**Comment by abunavas** [comment]
<p>Thank you <a class="mention" href="/ernest-phillips">@ernest-phillips</a></p>
<p>This is workaround. I suppose I shell ask in pandoc / LaTex forums, as well. Anyway, your answer is very useful.</p>
